package.json

```diff
@@ -34,5 +35,8 @@
   },
   "devDependencies": {
     "eslint": "2.8.0"
+  },
+  "dependencies": {
+    "eslint-plugin-react": "6.2.0"
```

Comment:
yeah, not too concerned with it, better than any of the alternative issues we could have from maintaining downstream packages.